docs/services/slam/_index.md

```diff
@@ -31,3 +31,203 @@ Integrated SLAM Libraries include:
 | Model | Description |
 | ----- | ----------- |
 | [`viam:slam:cartographer`](cartographer/) | [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data. |
+
+## API
+
+The SLAM Service supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[`GetPosition`](#getposition) | Get the current position of the specified source component in the point cloud SLAM map.
+[`GetPointCloudMap`](#getpointcloudmap) | Get the point cloud SLAM map.
+[`GetInternalState`](#getinternalstate) | Get the internal state of the SLAM algorithm required to continue mapping/localization.
+[`GetLatestMapInfo`](#getlatestmapinfo) | Get the timestamp of the last update to the point cloud SLAM map.
+
+{{% alert title="Tip" color="tip" %}}
+
+The following code examples assume that you have a robot configured with a SLAM Service called `"my_slam_service"`, and that you add the required code to connect to your robot and import any required packages at the top of your code file.
+Go to your robot's **Code sample** tab on the [Viam app](https://app.viam.com) for boilerplate code to connect to your robot.
+
+{{% /alert %}}
+
+### GetPosition
+
+Get the current position of the component the SLAM service is configured to source point cloud data from in the SLAM map as a [`Pose`](/internals/orientation-vector/).
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(spatialmath.Pose)](https://pkg.go.dev/go.viam.com/rdk@v0.3.0/spatialmath#Pose): A `Pose` representing the current position of the specified component.
+- [(string)](https://pkg.go.dev/builtin#string): The `"name"` of the component the SLAM service is configured to source point cloud data from.
+For example, a [camera](/components/camera/) named `"cam"`.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/slam).
+
+```go {class="line-numbers linkable-line-numbers"}
+slam_svc, err := slam.FromRobot(robot, "my_slam_service")
+
+// Get the current position of the specified source component in the SLAM map as a Pose.
+pos, name, err := slam_svc.GetPosition(context.Background())
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(Pose)](https://python.viam.dev/autoapi/viam/components/arm/index.html#viam.components.arm.Pose): A `Pose` representing the current position of the component the SLAM service is configured to source point cloud data from.
```

Comment:
@sguequierre , @jeremyrhyde is it right to link to components.arm.Pose, or is there a slam-specific link that would make more sense?